src/SFA.DAS.EmploymentCheck.Data.IntegrationTests/Repositories/DataCollectionsResponse/WhenGettingDataCollectionsResponseByEmploymentCheckIdWithMultipleRows.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.EmploymentCheck.Functions.Repositories;
+using System;
+using System.Threading.Tasks;
+using Models = SFA.DAS.EmploymentCheck.Functions.Application.Models;
+
+namespace SFA.DAS.EmploymentCheck.Data.IntegrationTests.Repositories.DataCollectionsResponse
+{
+    public class WhenGettingDataCollectionsResponseByEmploymentCheckIdWithMultipleRows
+        : RepositoryTestBase
+    {
+        private IDataCollectionsResponseRepository _sut;
+        private Models.DataCollectionsResponse _actual;
+
+        // As per the design, Ninos are being stored in the DataCollectionsResponse in a transaction after
+        // the related EmploymentCheck has been stored. The EmploymentCheck.ApprenticeshipId is not the primary
+        // key on the DataCollectionsResponse table so it's  possible for multiple EmploymentCheck.ApprenticeshipId's
+        // to be stored in the table (either due to issues with the manual loading of the records or re-running of functions)
+        // so on retrieval, we just get the lastest row if there are multiple rows.
```

Comment:
This comment isn't really needed, the test should be self explanatory. The name of this test accurately describes what it does, so it doesn't really need this explanation.

Reply:
Removed comment.